configurations/leap_s04_pipeline.py

```diff
@@ -0,0 +1,372 @@
+from core_data_modules.cleaners import swahili
+
+from src.pipeline_configuration_spec import *
+
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="leap_s04",
```

Comment:
It looks like this "wusc_leap_s04" in some other places, does it make sense to use that here too or stick with "leap_s04"? I have no strong opinions.